docs/install/_advanced.rst

```diff
@@ -8,44 +8,21 @@ Advanced installation options
 Generic installation (pip install)
 ----------------------------------
 
-Once Kolibri is released, you may install it as a standard package from PyPi using this command::
-
-    pip install kolibri --pre
-
-
-.. _ppa-installation:
-
-Debian/Ubuntu: Subscribe to updates through a PPA
--------------------------------------------------
-
-We maintain a `PPA on Launchpad <https://launchpad.net/~learningequality/+archive/ubuntu/kolibri>`__ and if you are connected to the internet, this will also give you automatic updates.
-
-On Ubuntu/Debian/Raspbian, do this::
-
-    sudo apt-get install software-properties-common python-software-properties
-    sudo add-apt-repository ppa:learningequality/kolibri
-    sudo apt-get update
-    sudo apt-get install kolibri
+You can install Kolibri as a standard package from PyPi (works on Mac, Windows, and Linux) using this command::
 
+    pip install kolibri
 
 .. _raspberry-pi-wifi:
 
 Raspberry Pi
 ------------
 
 .. note::
-  This section is pending instructions for creating a WI-FI hotspot. You can
-  install Kolibri on Raspbian following the above instructions for the PPA or
-  the ``pip install`` method.
+  This section is pending instructions for creating a WI-FI hotspot on Raspberry Pi. Follow the instructions above to install Kolibri on :ref:`Raspbian <rpi>`.
```

Comment:
There two Raspberry Pi sections (here and under Linux). Should probably just delete this one?

The fact that we eventually want to have instructions on setting up a wifi hotspot is not helpful